### ISSUE TITLE
Operator SDK with Helm : Can't attach to CockroachDB client

### ISSUE THREAD

**Issue by mamurak** [openshift-labs/learn-katacoda#1286]
**In which scenario did this bug occur?**

Operator SDK with Helm 

**At what step of the scenario did this happen?**

Steps 6, 8

**Describe the bug**

The scenario instructs the user to interact with CockroachDB using a temporary client container, which should be started via
`oc run -it --rm cockroach-client --image=cockroachdb/cockroach --restart=Never --command -- ./cockroach sql --insecure --host $COCKROACHDB_PUBLIC_SERVICE`
However, this command fails.

**Logs or error messages**

```
Error attaching, falling back to logs: unable to upgrade connection: Unauthorized
pod "cockroach-client" deleted
error: You must be logged in to the server (the server has asked for the client to provide credentials ( pods/log cockroach-client))
```

**Comment by madorn** [comment]
Please see https://github.com/openshift-labs/learn-katacoda/issues/1276

**Comment by madorn** [comment]
Fixed in #1290.

**Comment by jdob** [comment]
Closing per Matt's comment.
